src/home_robot/home_robot/agent/objectnav_agent/objectnav_agent.py

```diff
@@ -333,7 +333,7 @@ def act(self, obs: Observations) -> Tuple[DiscreteNavigationAction, Dict[str, An
             vis_inputs[0]["semantic_frame"] = obs.task_observations["semantic_frame"]
             vis_inputs[0]["closest_goal_map"] = closest_goal_map
             vis_inputs[0]["third_person_image"] = obs.third_person_image
-            vis_inputs[0]["short_term_goal"] = short_term_goal
+            vis_inputs[0]["short_term_goal"] = None
```

Comment:
why?

Reply:
Mainly because in some very rare case this was going a crash. For some reason, `goal` and `short_term_goal` were empty. This gets overriden by the agent arrow in visualizations, so it is never actually visualized. I let it stay because sometimes I use this to visualize.